client/src/components/EmailTemp.js

```diff
@@ -1,7 +1,195 @@
+
+    //Possible for later implementation
+    // handleChangeIllness(e) {
+    //     this.setState({
+    //         illness: e.target.illness
+    //     });
+    // }
+
+    handleChangePhone(e) {
+        this.setState({
+            phone: e.target.value
+        });
+    }
+
+    handleChangeValue = (e, {value}) => {
+        this.setState({
+            val: value
+        })
+    }
+
+    template (value) {
+        if(value == 1) {
+            return (
+                <Container text>
+                    <Header as='h2'>Your Email To Your Social Worker:</Header>
+                <p>
+                    Hello. My name is {this.state.name} {this.state.lName}. I
+                    wanted to reach out to you for assistance. I currently live
+                    in a food desert - I usually stay in the {this.state.city}
+                    area. I have chronic illnesses,
+                    and take medication that requires me to have regular meals.
+                    Given the fact that I live in a food desert, I would like
+                    to find methods/places where I can get food on a regular
+                    basis. I would love to speak with you about any
+                    opportunities you think would be helpful for me.
+                    You can contact me by phone at {this.state.phone}. <br/>
+                    Thank you, <br/>
+                    {this.state.name} {this.state.lName}
+                </p>
+                </Container>
+            );
+        } else if (value == 2) {
+            return (
+                <Container text>
+                    <Header as='h2'>Your Email To Your Doctor:</Header>
+                    <p>
+                        Hello. My name is {this.state.name} {this.state.lName}. I
+                        wanted to reach out to you for assistance in connecting with
+                        a social worker. I currently live in a food desert - I
+                        usually stay in the {this.state.city} area. I have chronic
+                        illnesses and take medication that requires me to have
+                        regular meals. Given the fact that I live in a food desert,
+                        I would like to find methods/places where I can get food on
+                        a regular basis. I would love to speak with you about any
+                        opportunities you think would be helpful for me. You can
+                        contact me by phone at {this.state.phone}. <br/>
+
+                        Thank you, <br/>
+
+                        {this.state.name} {this.state.lName}
+                    </p>
+                </Container>
+            );
+        } else {
+            return (
+                <div>
+
+                    Please select a template
+                </div>
+            );
+        }
+    }
+
     render() {
-        return <h1>EmailTemp</h1>
+        return (
+            <div>
+                <h1>EmailTemp</h1>
```

Comment:
We should change this header to "Ask for Help" or something descriptive like that